build_image.py

```diff
@@ -194,6 +194,8 @@ def create_dockerfile_base(config: OfrakImageConfig) -> str:
 
     dockerfile_base_parts += [
         "FROM python:3.7-bullseye@sha256:338ead05c1a0aa8bd8fcba8e4dbbe2afd0283b4732fd30cf9b3bfcfcbc4affab",
+        "RUN apt-get -y update",
```

Comment:
The drawback of this proposed change is that the update step is cached separate from the install step.

[Dockerfile best practices](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run) generally recommends running update with the install step. The `Dockerstub` approach we have taken generally should group each `apt-get install` in each `Dockerstub` into one step (the exception here is `ofrak-patch-maker`, where we've maintained different steps to make it clear which dependency matches each toolchain).

(And yes, we probably should be running `rm -rf /var/lib/apt/lists/*` to keep the image a little bit smaller).

Reply:
Makes sense - I reverted this change, and instead made sure that each `apt-get install` is preceded by `apt-get update`